recipe/meta.yaml

```diff
@@ -37,7 +34,7 @@ requirements:
     - regex !=2019.12.17
     - requests
     - safetensors >=0.4.1
-    - tokenizers >=0.14,<0.19
+    - tokenizers >=0.19,<0.20
     - tqdm >=4.27
```

Comment:
the latest feedstock version gives a [bunch more packages in run_constrained](https://github.com/AnacondaRecipes/transformers-feedstock/blob/3fd34517043959cbb8981b7921e1f9c022696000/recipe/meta.yaml#L39) to cover the [long list of deps upstream](https://github.com/huggingface/transformers/blob/4fdf58afb72b0754da30037fc800b6044e7d9c99/setup.py#L97). You could consider including this here, adapting for v4.40.2

Reply:
Thank you for the suggestion! Fixed.